src/lib/api-v1.ts

```diff
@@ -115,13 +115,14 @@ export default class ApiV1 {
   }: Props): Promise<AxiosResponse<ActivateWorkflowResp>> {
     const params = prune({
       environment: flags.environment,
+    });
+    const data = prune({
       recipients: flags.recipients,
       tenant: flags.tenant,
       data: flags.data,
       actor: flags.actor,
     });
-
-    return this.put(`/workflows/${args.workflowKey}/run`, {}, { params });
+    return this.put(`/workflows/${args.workflowKey}/run`, data, { params });
```

Comment:
Changed this so we sent the data on the `body` of the request, but keep the `environment` on the `queryParams` as we do on other requests (i.e `upsertWorkflow`).

Reply:
That's perfect, thanks franco!